lib/Controller/RecordingController.php

```diff
@@ -66,4 +80,34 @@ public function stopRecording(): DataResponse {
 		}
 		return new DataResponse();
 	}
+
+	/**
+	 * @PublicPage
+	 * @RequireRoom
+	 * @BruteForceProtection(action=talkSipBridgeSecret)
+	 *
+	 * @return DataResponse
+	 */
+	public function store(string $owner): DataResponse {
+		try {
+			$random = $this->request->getHeader('TALK_SIPBRIDGE_RANDOM');
+			$checksum = $this->request->getHeader('TALK_SIPBRIDGE_CHECKSUM');
+			$secret = $this->talkConfig->getSIPSharedSecret();
+			if (!$this->SIPBridgeService->validateSIPBridgeRequest($random, $checksum, $secret, $this->room->getToken())) {
```

Comment:
The upload is done by the HPB, not the SIP bridge, right? @danxuliu 
So I guess we need to use the `getSignalingSecret` here in the end.
However that shows that the method is independent from "SIP" and should maybe have a neutral name and home (HelperService ?)